<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileInputWindow</class>
 <widget class="QMainWindow" name="FileInputWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="resultTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>назад</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectAndAnalyzeButton">
        <property name="text">
         <string>анализировать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportButton">
        <property name="text">
         <string>экспорт</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearCacheButton">
        <property name="text">
         <string>очистить кэш</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
